System.IO.Abstractions/System.IO.Abstractions.csproj

```diff
@@ -21,6 +21,21 @@
         <SignAssembly>True</SignAssembly>
     </PropertyGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="System.IO.FileSystem.AccessControl">
+      <Version>4.4.0</Version>
```

Comment:
I propose to use version 4.5.0 as it removes DLLs for many targets, but offers single netstandard2.0 for all these platforms added since 4.3.0. It's good that you haven't used 4.3.0 which won't work on Android

Reply:
Changed it to use 4.5.0